scripts/read_secureboot.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
I think this can just be inlined into the zen_workaround script instead of being a separate script.